source/algorithms.tex

```diff
@@ -6322,8 +6314,6 @@
 the \oldconcept{MoveConstructible} (\tref{cpp17.moveconstructible}) and
 \oldconcept{\-Move\-Assignable} (\tref{cpp17.moveassignable}) requirements.
 
-\pnum
-\expects
```

Comment:
Not really an order correction, but a merge. Shouldn't it become a list? This is how it looks like (single paragraph number, multiple grammatical paragraphs, treating the two different matters being merged):
![1593821626](https://user-images.githubusercontent.com/21071787/86501781-4cbf2600-bd6a-11ea-9cb1-d1449a65555f.png)

Reply:
Similar changes in this PR didn't dropped the `\expects` and not the `\pnum`, so perhaps that would do here.

Reply:
Fixed.